tests/extensions/tabular/test_renderer.py

```diff
@@ -46,6 +46,25 @@ def invalid_xlsx_file_path():
 
 
 @pytest.fixture
+def invalid_mat73_file_path():
+    return os.path.join(os.path.dirname(os.path.abspath(__file__)), 'files', 'invalidVer73.mat')
```

Comment:
While you're in here, how about saving `os.path.dirname(os.path.abspath(__file__)), 'files'` into an intermediate variable, so save all the c-and-p in the fixtures?